apps/communications/contacts/js/contacts.js

```diff
@@ -219,7 +214,9 @@ var Contacts = (function() {
 
     // We need to return the promise here for testing purposes
     return addAsyncScripts().then(() => {
-      checkUrl();
+      Loader.utility('Confirm', function viewLoaded() {
```

Comment:
Why do you need to load `Confirm` before checkUrl? Could we load this within `checkUrl` if needed just in the case we use it?